var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -33,6 +33,8 @@ class PyScipy(PythonPackage):
     homepage = "http://www.scipy.org/"
     url = "https://pypi.io/packages/source/s/scipy/scipy-0.18.1.tar.gz"
 
+    version('0.19.0', '91b8396231eec780222a57703d3ec550',
+            url="https://pypi.io/packages/source/s/scipy/scipy-0.19.0.zip")
```

Comment:
if they changed to `.zip`, later we could add `url` for version to distinguish between `:0.18.1` and newer.